LICENSE-3rdparty.csv

```diff
@@ -777,12 +776,14 @@ core,go.opentelemetry.io/collector/config/confighttp,Apache-2.0,Copyright The Op
 core,go.opentelemetry.io/collector/config/confignet,Apache-2.0,Copyright The OpenTelemetry Authors
 core,go.opentelemetry.io/collector/config/configtelemetry,Apache-2.0,Copyright The OpenTelemetry Authors
 core,go.opentelemetry.io/collector/config/configtls,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/experimental/configsource,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/mapconverter/expandmapconverter,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/mapconverter/overwritepropertiesmapconverter,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/mapprovider/envmapprovider,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/mapprovider/filemapprovider,Apache-2.0,Copyright The OpenTelemetry Authors
-core,go.opentelemetry.io/collector/config/mapprovider/yamlmapprovider,Apache-2.0,Copyright The OpenTelemetry Authors
+core,go.opentelemetry.io/collector/confmap,Apache-2.0,Copyright The OpenTelemetry Authors
```

Comment:
This file is really messed up. Several lines have extra columns....

Reply:
Here is the rendered version, if it helps: https://github.com/DataDog/datadog-agent/blob/26ca746ff3ba16aeca6dfa95721d63202be8f534/LICENSE-3rdparty.csv The file is updated by a script (both in this PR and everywhere else), so I would expect the format to be correct. What lines seem wrong?

Reply:
Ah, I see now. My brain was thinking the pipes were column endings - not enough coffee this morning! Some of the Copyright columns are especially long with lots of contributors being attributed (for example :1094) but I suppose that's appropriate for the license types on those lines.